iModelJsNodeAddon/IModelJsNative.cpp

```diff
@@ -394,7 +394,10 @@ struct NativeECDb : BeObjectWrap<NativeECDb>, SQLiteOps {
         Napi::HandleScope scope(val.Env());
         return val.As<Napi::Object>().InstanceOf(Constructor().Value());
     }
-
+    void RequireDbIsOpen(NapiInfoCR info) {

```

Comment:
I don't understand why this is necessary. It's already implemented in SQLiteOps.

Reply:
We have unsafe calls like the following. An ECDb/iModel that is closed will crash if the following method is called.
```cpp
SchemaManager const& ECDb::Schemas() const { return m_pimpl->Schemas(); }
```

Reply:
I understand why you need to call it, but it is already implemented in SQLiteOps. You can just delete this implementation,

Reply:
Fixed.